common/app/views/fragments/footer.scala.html

```diff
@@ -230,6 +230,9 @@
                         <li class="colophon__item"><a data-link-name="international : footer : work for us" href="https://workforus.theguardian.com">
                             work for us</a>
                         </li>
+                        <li class="colophon__item"><a data-link-name="uk : footer : contact us" href="@LinkTo {/help/contact-us}">
```

Comment:
Check the data link name. Its still `uk` and I think it should be `international` like above